src/main/java/frc/robot/Robot.java

```diff
@@ -32,23 +29,25 @@ public void robotInit() {
             Logger.addDataReceiver(new WPILOGWriter()); // Log to a USB stick ("/U/logs")
             Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
             new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
-        } else if (Constants.currentMode == Constants.Mode.SIM) {
```

Comment:
Why are we getting rid of the distinction between sim and replay modes?

Reply:
Earlier you mentioned wanting to use robot.isReal in place of Constants.Mode, but I plan to change it back because I've now realized the use of replay mode. I'll undo those changes.

Reply:
Added it back in 👍